src/supported-modules.h

```diff
@@ -56,6 +57,7 @@ extern pam_module_t mod_pam_u2f;
 
 pam_module_t *common_module_list[] = {
   &mod_pam_himmelblau,
+  &mod_pam_kanidm,
```

Comment:
The module list should be in alphabetical order.
Sorry, I missed that for himmelblau.

Reply:
Ah, my mistake.

Reply:
Done, I have fix the module ordering in both supported-modules.h and the pam-config.8.xml.